pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +213,16 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("persistentvolumes", "persistentvolumes/status", "storageclasses").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("events", "limitranges", "namespaces", "namespaces/status",
```

Comment:
@screeley44 @liggitt  I guess it's possible a storage admin would create the PVCs.If we are going to let the storage admin set the quotas, these are project scoped as well, so I don't see what would be the point of leaving pvcs out.